OSDOCS-17014-4: INSTALL-8: OpenStack (#115430)
openshift/openshift-docs: installing/installing_openstack/deploying-openstack-with-rootVolume-etcd-on-local-disk.adoc

toc::[]

[role="_abstract"]

> As a post-installation operation, you can resolve and prevent performance issues of your {rh-openstack-first} installation by moving etcd from a root volume (provided by {rh-openstack} Cinder) to a dedicated ephemeral local disk.

Reviewer: ```suggestion
After installation, you can resolve and prevent performance issues of your {rh-openstack-first} installation by moving etcd from a root volume (provided by {rh-openstack} Cinder) to a dedicated ephemeral local disk.
```
IMHO you could edit this down for minimalism; not remotely a merge blocker tho

Author: :heavy_check_mark:
